projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/experiment/ExperimentalDesign.java

```diff
@@ -0,0 +1,118 @@
+package life.qbic.projectmanagement.domain.project.experiment;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Supplier;
+import life.qbic.application.commons.Result;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public class ExperimentalDesign {
+
+  private final List<ExperimentalVariable<ExperimentalValue>> experimentalVariables;
+  private final List<Condition> conditions;
+  private List<SampleGroup> sampleGroups;
+
+  public ExperimentalDesign() {
+    experimentalVariables = new ArrayList<>();
+    conditions = new ArrayList<>();
+    sampleGroups = new ArrayList<>();
+  }
+
+  public Result<ExperimentalVariable<ExperimentalValue>, Exception> createExperimentalVariable(
+      String variableName, ExperimentalValue... levels) {
+    return addExperimentalVariableOrElse(new ExperimentalVariable<>(variableName, levels),
+        () -> Result.failure(new ExperimentalVariableExistsException(
+            String.format(
+                "A variable with name '%s' is already part of the experimental design. Variable names need to be unique within the design.",
+                variableName))));
+  }
+
+  public Result<Condition, Exception> createCondition(
+      VariableLevel<ExperimentalValue>... variableLevels) {
+    return addToDesignOrElse(Condition.create(variableLevels), () -> Result.failure(
+        new UnknownExperimentalVariableException(
+            "At least one experiment variable is not defined in the design.")));
```

Comment:
This message might not match the error. If the condition checks for at least one variable and none is given here, this leads to a wrong error message.